release/crankshaft--0.0.1.sql

```diff
@@ -137,6 +137,33 @@ BEGIN
 END;
 $$
 LANGUAGE plpgsql VOLATILE;
+CREATE OR REPLACE FUNCTION
+  cdb_create_segment (
+      segment_name TEXT,
+      table_name TEXT,
+  	  column_name TEXT,
+      geoid_column TEXT DEFAULT 'geoid',
+      census_table TEXT DEFAULT 'block_groups'
```

Comment:
@ohasselblad @stuartlynn a released version shall never be touched. that's the intent of the directory release.

No worries, I take this fully in charge, but please mind the next time.